### ISSUE TITLE
Consider role in the label rule

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2458]
The following code snippets should not fail the `label` rule. This can be fixed by adding the role-none and role-presentation checks to that rule (as well as a few integration tests).

```html
<input role="none" disabled />
```

```html
<input role="presentation" disabled />
```

**Comment by padmavemulapati** [comment]
Verified in the latest code - `label` is passing when   role-none and role-presentation  `<input role="none" disabled />` and also 
`<input role="presentation" disabled />`
and at the same time failure is there for `aria-allowed-attribute` , which is expected.
![image](https://user-images.githubusercontent.com/47409579/91040952-c4078c80-e62c-11ea-93d9-eea8900021b9.png)

**Comment by straker** [comment]
No docs required
